pkg/auth/authorizer/abac/abac_test.go

```diff
@@ -41,7 +41,7 @@ func TestOneLineFileNoNewLine(t *testing.T) {
 
 func TestTwoLineFile(t *testing.T) {
 	_, err := newWithContents(t, `{"user":"scheduler",  "readonly": true, "resource": "pods"}
-{"user":"scheduler",  "readonly": true, "resource": "services"}
+{"user":"scheduler",  "readonly": true, "kind": "services"}
```

Comment:
yup, just a merge relict, i think. (surprisingly doesn't seem to affect the tests), but in the 1.0 branch "ns" and "kind" are used everywhere and the unit tests don't complain. 

speaking of which: as the feature is additive, is there a chance to apply this to the 1.0 branch?